app/src/main/java/com/tip/lunchbox/model/Location.java

```diff
@@ -0,0 +1,55 @@
+package com.tip.lunchbox.model;
```

Comment:
@epicadk I think you should create _setter_ methods also as we may need them in future